tests/helpers/strings.spec.js

```diff
@@ -112,4 +112,134 @@ describe('strings', () => {
+        it('uppercase should return uppercase value of a string param', () => {
+            expect(strings.uppercase('hello world!')).toEqual('HELLO WORLD!');
+        });
+
+        it('uppercase should work as expected after compilation (Basic Support)', () => {
+            var template = compile('{{uppercase val}}');
+            var obj = {
+                val: 'just wow!!!'
+            };
+
+            expect(template(obj)).toEqual('JUST WOW!!!');
+        });
+    });
+
+    describe('first', () => {
+        it('first should return first element of an array(string)', () => {
+            expect(strings.first(['David', 'Miller', 'Jones'])).toEqual('David');
+        });
+
+        it('first should work as expected after compilation (Basic Supprt)', () => {
+            var template = compile('{{first fullName}}');
+            var obj = {
+                fullName: [
+                    'David',
+                    'Miller',
+                    'Jones'
+                ]
+            };
+
+            expect(template(obj)).toEqual('David');
+        });
+    });
+
+    describe('last', ()=> {
+        it('last should return last element of an array(string)', () => {
+            expect(strings.last(['David', 'Miller', 'Jones'])).toEqual('Jones');
+        });
+
+        it('last should work as expected after compilation (Basic Supprt)', () => {
+            var template = compile('{{last fullName}}');
+            var obj = {
+                fullName: [
+                    'David',
+                    'Miller',
+                    'Jones'
+                ]
+            };
+
+            expect(template(obj)).toEqual('Jones');
+        });
+    });
+
+
+    describe('concat', () => {
+        it('concat should return concatenation of all param values(string)', () => {
+            expect(strings.concat('hello', ' ', 'world', '!!!')).toEqual('hello world!!!');
+        });
+
+        it('concat should return concatenation of all param values(string and integer)', () => {
+            expect(strings.concat('I have got', ' ', 4, ' ', 'apples.')).toEqual('I have got 4 apples.');
+        });
+
+        it('concat should work as expected for string params after compilation (Basic Support)', () => {
+            var template = compile('{{concat first " " middle " " last}}');
+            var name = {
+                first: 'David',
+                middle: 'Miller',
+                last: 'Jones'
+            };
+
+            expect(template(name)).toEqual('David Miller Jones');
+        });
+
+        it('concat should work as expected for string and integer params after compilation (Basic Support)', () => {
+            var template = compile('{{concat s1 amount s2}}');
+            var sentence = {
+                s1: 'I have got ',
+                amount: 4,
+                s2: ' apples.'
+            };
+
+            expect(template(sentence)).toEqual('I have got 4 apples.');
+        });
+    });
+
+    /* join */
```

Comment:
no need of this anymore as `describe('join', () => {` **describes** it better :wink: